contentcuration/contentcuration/frontend/channelEdit/components/edit/EditModal.vue

```diff
@@ -489,13 +489,19 @@
       },
       createNodesFromUploads(fileUploads) {
         fileUploads.forEach((file, index) => {
-          const title = file.original_filename
+          let title = file.original_filename
             .split('.')
             .slice(0, -1)
             .join('.');
+          if (title === undefined) {
+            title = file.metadata.title;
+          }
+          const language = file.metadata.language !== undefined ? file.metadata.language : null;
+          const author = file.metadata.authors !== undefined ? file.metadata.author : null;
+          const license = file.metadata.license !== undefined ? file.metadata.license : null;
           this.createNode(
             FormatPresets.has(file.preset) && FormatPresets.get(file.preset).kind_id,
-            { title }
+            { title, language, author, license }
```

Comment:
I think this can be more concisely managed using destructuring, e.g.:

```
const properties = {
  title: file.original_filename,
  ...file.metadata,
};
```

This will only take defined properties from the metadata object and put them into the properties, so we are not passing null values (which means that any defaults will be properly applied by our createNode function).

Reply:
Yes destructuring can manage concisely Thank You.